editor/editor_inspector.cpp

```diff
@@ -778,10 +778,20 @@ Control *EditorProperty::make_custom_tooltip(const String &p_text) const {
 	help_bit->add_style_override("panel", get_stylebox("panel", "TooltipPanel"));
 	help_bit->get_rich_text()->set_fixed_size_to_width(360 * EDSCALE);
 
-	String text = TTR("Property:") + " [u][b]" + p_text.get_slice("::", 0) + "[/b][/u]\n";
-	text += p_text.get_slice("::", 1).strip_edges();
-	help_bit->set_text(text);
-	help_bit->call_deferred("set_text", text); //hack so it uses proper theme once inside scene
+	PackedStringArray slices = p_text.split("::", false);
+	if (!slices.empty()) {
+		String property_name = slices[0].strip_edges();
+		String text = TTR("Property:") + " [u][b]" + property_name + "[/b][/u]";
+
+		if (slices.size() > 1) {
+			String property_doc = slices[1].strip_edges();
+			if (property_name != property_doc) {
```

Comment:
Would this still happened? I thought you did that before because if the second slice is missing it would return the same output as the first slice. If not, then my refactoring suggestion was maybe not particularly useful. It doesn't hurt though.

Reply:
`p_text` can be either those two things:
- `property_name::"The property's description."`
- `property_name::property_name`

Reply:
I guess it could be refactored to give an empty slice if there's no description, and by excluding empty slices on split, you'd only get one slice. But it's maybe not so important.

Reply:
That's how it's done in this commit.

Reply:
I mean the code that generates `p_text` could be changed. But that might affect other stuff, not sure how widely used it is.